source/manual/alerts/email-alert-api-unprocessed-subscription-contents.html.md

```diff
@@ -0,0 +1,36 @@
+---
+owner_slack: "#govuk-2ndline"
+title: 'Email Alert API: Unprocessed subscription contents'
+section: Icinga alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-01-07
+review_in: 6 months
+---
+
+This means there there are subscription contents being created without emails
+associated with them, and implies that emails aren't being sent out.
+
+### Icinga checks
+
+* `warning` - `subscription_contents` created over 10 minutes ago (or 35 minutes
+  ago if during publishing times)
+* `critical` - `subscription_contents` created over 15 minutes ago (or 50 minutes
+  during publishing times)
+
+See the [subscription_contents_worker][subscription-content-worker]
+for more information.
+
+### Check which subscription contents this affects
+
+```ruby
+SubscriptionContent.where("subscription_contents.created_at < ?", 10.minutes.ago).where(email: nil).joins(:subscription).merge(Subscription.active)
+```
+
+Check the count, then run the above query again to see if the count has
+decreased. If it's decreasing, then it means that emails are going out and
+there's probably a lot being processed. You can also check the
```

Comment:
If the count is not increasing, what should people do to solve this?

Reply:
I hit the wrong button. 🤦‍♂

Reply:
Thanks @issyl0 ,I have added a suggestion on what to do if it's not decreasing. It's only a guess but at least a starting point to investigate!